helm_deploy/hmpps-interventions-service/templates/deployment-performance-report.yaml

```diff
@@ -34,7 +34,7 @@ spec:
                         - {{ template "app.name" . }}
                 topologyKey: kubernetes.io/hostname
       containers:
-        - name: hmpps-interventions-service
+        - name: provider-report-api
```

Comment:
Or we could copy the deployment name, eg. `performance-report` -- maybe that's less surprising?